packages/leavitt-elements/src/leavitt-person-group-select.ts

```diff
@@ -0,0 +1,322 @@
+import { css, customElement, html, LitElement, property, query } from 'lit-element';
+import '@leavittsoftware/profile-picture';
+import '@material/mwc-textfield';
+import '@material/mwc-list/mwc-list-item.js';
+import '@material/mwc-linear-progress';
+import '@material/mwc-menu';
+
+import { ifDefined } from 'lit-html/directives/if-defined';
+import { Menu } from '@material/mwc-menu';
+import { TextField } from '@material/mwc-textfield';
+import { TitaniumSnackbarSingleton as AppSnackbar } from '@leavittsoftware/titanium-snackbar/lib/titanium-snackbar';
+
+import { getSearchTokens, Debouncer, LoadWhile } from '@leavittsoftware/titanium-helpers';
+import Api2ServiceMixin from '@leavittsoftware/api-service/lib/api2-service';
+import { Person as CorePerson, PeopleGroup as CorePeopleGroup } from '@leavittsoftware/lg-core-typescript/lg.core';
+import { peopleGroupIcons } from './people-group-icons';
+
+export class LeavittPersonGroupSelectSelectedEvent extends Event {
+  static eventType = 'selected';
+  person: Partial<Person> | null;
```

Comment:
Can we make the types of the properties in the event CorePerson and CorePeopleGroup? Otherwise every consumer of this will have to cast to their Person/PeopleGroup when they use the value.

Reply:
Updated.  Although I think the types are interchangeable in that direction because typescript knows its an inter-section type of the Core type.